src/app/pages/blog/blog-context.tsx

```diff
@@ -0,0 +1,212 @@
+import React from 'react';
```

Comment:
View as diffs [here](https://github.com/openstax/os-webview/pull/2676/commits/5dc3fad04120d1b570afffb73036dccd216c804e#diff-449b99e466a24c7ddf6d31b1f58e0742ee9968030eb77ce43b7e9e777d7fb0d7). For this one, it's just a bunch of types added. A big block of them at the top, and others peppered throughout.